docs/deployment/universal-profile.md

```diff
@@ -349,22 +328,8 @@ lspFactory.UniversalProfile.deploy({...}, {
 })
 ```
 
-### Using a Custom Address
-
-When using proxy deployment you can specify the base contract address by passing the `version` parameter. This allows you to deploy a specific contract implementation by using a custom base contract you have deployed.
-
-Any base contract address that developers pass here must adhere to the relevant LSP contract standard it is being used for.
-
-```javascript title="Deploying a Universal Profile using a custom ERC725Account base contract implementation"
-lspFactory.UniversalProfile.deploy({...}, {
-    ERC725Account: {
-        version: '0x00b1d454Eb5d917253FD6cb4D5560dEC30b0960c',
-    },
-})
-```
-
 :::info
-The `UniversalReceiverDelegate` contract does not use proxy deployment by default. If an address is passed to the `UniversalReceiverDelegate` `version` parameter and `deployProxy` is not set to `true`, LSPFactory will set the provided address directly on the ERC725Account as the [LSP1UniversalReceiverDelegate key](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md#lsp1universalreceiverdelegate) and given the `SETDATA` LSP6 permission. You can read more in the [section above](./universal-profile#universal-receiver-delegate-proxy-deployment).
+The `UniversalReceiverDelegate` contract does not use proxy deployment by default. If an address is passed to the `LSP1UniversalReceiverDelegate` `version` parameter and `deployProxy` is not set to `true`, LSPFactory will set the provided address directly on the ERC725Account as the [LSP1UniversalReceiverDelegate key](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md#lsp1universalreceiverdelegate) and the `SETDATA` LSP6 permission.
```

Comment:
you can add this link on SETDATA LSP6 permisison maybe:

https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager#permissions